lib.typ

```diff
@@ -3,23 +3,41 @@
   markdown,
   smart-punctuation: true,
   blockquote: none,
+  mitex: none,
   h1-level: 1,
   raw-typst: true,
   scope: (:),
   show-source: false,
 ) = {
-  let options = 0;
-  if smart-punctuation { options += 0b00000001; }
-  if blockquote != none { options += 0b00000010; }
-  if raw-typst { options += 0b00000100; }
+  if type(markdown) == content and markdown.has("text") {
+    markdown = markdown.text
+  }
+  let options = 0
+  if smart-punctuation {
+    options += 0b00000001
+  }
+  if blockquote != none {
+    options += 0b00000010
+  }
+  if raw-typst {
+    options += 0b00000100
+  }
+  if mitex != none {
+    options += 0b00001000
+    scope += (inlinemath: mitex.with(block: false), displaymath: mitex.with(block: true))
+  }
   let rendered = str(_p.render(bytes(markdown), bytes((options, h1-level))))
   if show-source {
     raw(rendered, block: true, lang: "typ")
   } else {
-    eval(rendered, mode: "markup", scope: (
-      blockquote: blockquote,
-      image: (..args) => image(..args),
-      ..scope,
-    ))
+    eval(
+      rendered,
+      mode: "markup",
+      scope: (
+        blockquote: blockquote,
```

Comment:
Can this be moved to a `scope +=` statement in the `if blockquote != none` block, for consistency?